ReportingServicesTools/Functions/CatalogItems/Update-RsSubscription.ps1

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)

```

Comment:
You will also need to update the manifest to include the new function
[https://github.com/Microsoft/ReportingServicesTools/blob/master/ReportingServicesTools/ReportingServicesTools.psd1](https://github.com/Microsoft/ReportingServicesTools/blob/master/ReportingServicesTools/ReportingServicesTools.psd1)

Reply:
added Update-RsSubscription to the manifest